docs/contents.md

```diff
@@ -0,0 +1,38 @@
+| [Home](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/README.md) |
+|--------------------------------------------|
+
+# Contents
+
+The **File Content Extraction** solution pack contains the following resources.
+
+## Connectors
+
+|**Name**|**Description**|
+| :- | :- |
+| Exchange | The Exchange connector provides a robust, platform-independent, and simple interface for communicating with Microsoft Exchange 2007-2016 Server or Office 365 using Exchange Web Services (EWS). |
+| File Content Extraction | The connector uses is used to extract content, metadata and artifacts from over 1500 file types such as MS Office, PDF...etc |

```

Comment:
Please improve the File Content Extraction description sentence 

`The connector uses is used to extract content`

Reply:
modified